client/process.go

```diff
@@ -134,16 +134,16 @@ func (config *Config) DebugGCS() {
 	if cmd == "" {
 		cmd = `sh -c "`
 		cmd += debugCommand("kill -10 `pidof gcs`") // SIGUSR1 for stackdump
-		cmd += debugCommand("ls -l /tmp")
-		cmd += debugCommand("cat /tmp/gcs.log")
-		cmd += debugCommand("cat /tmp/gcs/gcs-stacks*")
-		cmd += debugCommand("cat /tmp/gcs/paniclog*")
-		cmd += debugCommand("ls -l /tmp/gcs")
-		cmd += debugCommand("ls -l /tmp/gcs/*")
-		cmd += debugCommand("cat /tmp/gcs/*/config.json")
+		cmd += debugCommand("ls -l /run")
```

Comment:
Just remove these changes since client is going away anyways.

Reply:
Done.